packages/web3-eth/test/integration/defaults.test.ts

```diff
@@ -1011,7 +1015,7 @@ describe('defaults', () => {
 			});
 			expect(newBuilderMock).toHaveBeenCalled();
 		});
-		it('transactionTypeParser', () => {
+		it('transactionTypeParser', async () => {
```

Comment:
```suggestion
		it('transactionTypeParser', () => {
```
as change for looking block in this function rollbacked?